packages/nc-gui/components/template/Editor.vue

```diff
@@ -501,12 +501,16 @@ async function importTemplate() {
             }
           }
         }
-        const createdTable = await $api.base.tableCreate(project?.value?.id as string, baseId as string, {
-          table_name: table.table_name,
-          // leave title empty to get a generated one based on table_name
-          title: '',
-          columns: table.columns || [],
-        })
+        const createdTable = await $api.base.tableCreate(
+          project?.value?.id as string,
+          (baseId || project?.value?.bases?.[0].id)!,
```

Comment:
The `project` will be always defined so you can replace `project?.value?` with `project.value?`